packages/rpc-core/src/rpc-subscriptions/index.ts

```diff
@@ -3,33 +3,48 @@ import { IRpcSubscriptionsApi, RpcSubscription } from '@solana/rpc-transport/dis
 import { patchParamsForSolanaLabsRpc } from '../params-patcher';
 import { patchResponseForSolanaLabsRpcSubscriptions } from '../response-patcher';
 import { SlotNotificationsApi } from './slot-notifications';
+import { SlotsUpdatesNotificationsApi } from './slots-updates-notifications';
 
 type Config = Readonly<{
     onIntegerOverflow?: (methodName: string, keyPath: (number | string)[], value: bigint) => void;
 }>;
 
 export type SolanaRpcSubscriptions = SlotNotificationsApi;
+export type SolanaRpcSubscriptionsUnstable = SlotsUpdatesNotificationsApi;
 
-export function createSolanaRpcSubscriptionsApi(config?: Config): IRpcSubscriptionsApi<SolanaRpcSubscriptions> {
-    return new Proxy({} as IRpcSubscriptionsApi<SolanaRpcSubscriptions>, {
+export function createSolanaRpcSubscriptionsApi(
+    config?: Config
+): IRpcSubscriptionsApi<SolanaRpcSubscriptions & SolanaRpcSubscriptionsUnstable> {
+    return new Proxy({} as IRpcSubscriptionsApi<SolanaRpcSubscriptions & SolanaRpcSubscriptionsUnstable>, {
         defineProperty() {
             return false;
         },
         deleteProperty() {
             return false;
         },
-        get<TNotificationName extends keyof IRpcSubscriptionsApi<SolanaRpcSubscriptions>>(
-            ...args: Parameters<NonNullable<ProxyHandler<IRpcSubscriptionsApi<SolanaRpcSubscriptions>>['get']>>
+        get<
+            TNotificationName extends keyof IRpcSubscriptionsApi<
+                SolanaRpcSubscriptions & SolanaRpcSubscriptionsUnstable
+            >
+        >(
+            ...args: Parameters<
+                NonNullable<
+                    ProxyHandler<IRpcSubscriptionsApi<SolanaRpcSubscriptions & SolanaRpcSubscriptionsUnstable>>['get']
+                >
+            >
         ) {
             const [_, p] = args;
-            const notificationName = p.toString() as keyof SolanaRpcSubscriptions;
+            const notificationName = p.toString() as keyof (SolanaRpcSubscriptions & SolanaRpcSubscriptionsUnstable);
             return function (
                 ...rawParams: Parameters<
-                    SolanaRpcSubscriptions[TNotificationName] extends CallableFunction
-                        ? SolanaRpcSubscriptions[TNotificationName]
+                    (SolanaRpcSubscriptions &
+                        SolanaRpcSubscriptionsUnstable)[TNotificationName] extends CallableFunction
+                        ? (SolanaRpcSubscriptions & SolanaRpcSubscriptionsUnstable)[TNotificationName]
                         : never
                 >
-            ): RpcSubscription<ReturnType<SolanaRpcSubscriptions[TNotificationName]>> {
+            ): RpcSubscription<
+                ReturnType<(SolanaRpcSubscriptions & SolanaRpcSubscriptionsUnstable)[TNotificationName]>
+            > {
```

Comment:
Wait, I don't think we need this right? Instead, in your `_UNSTABLE` method you just _force_ the type of this thing to be `IRpcSubscriptionsApi<SolanaRpcSubscriptions & SolanaRpcSubscriptionsUnstable>` because we know it's compatible.

I general, I want someone who uses this directly to get none of the unstable methods. The only way to get the unstable stuff is to override the types (or use the helper you've created that overrides them for you).

Reply:
OK I'll check that out but I thought I was getting a Type Error when trying to provide both. But maybe just a straight up `as` cast is what you meant? I'll check it out.

Reply:
Cool, I think I found a good solution that works better than what I had. I just introduced a forced type from `createSolanaSubscriptionsApi_UNSTABLE()` so you can pass that as an `api` arg in the `createJsonSubscriptionsRpc()` function.

Should be good to go.